library/redis

```diff
@@ -1,59 +1,47 @@
-# this file is generated via https://github.com/docker-library/redis/blob/016e0d738031c00bb6cfa4a433e763413e14b757/generate-stackbrew-library.sh
+# this file is generated via https://github.com/docker-library/redis/blob/f610cf8906e0e1f6b4fbe37584e2d6051a3f6fc0/generate-stackbrew-library.sh
 
 Maintainers: Tianon Gravi <admwiggin@gmail.com> (@tianon),
              Joseph Ferguson <yosifkit@gmail.com> (@yosifkit)
 GitRepo: https://github.com/docker-library/redis.git
 
-Tags: 3.0.7, 3.0
-Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le, s390x
-GitCommit: fa0e115934f352cbfa8d0d471b43a18e865e28b3
-Directory: 3.0
-
-Tags: 3.0.7-32bit, 3.0-32bit
-Architectures: amd64
-GitCommit: fa0e115934f352cbfa8d0d471b43a18e865e28b3
-Directory: 3.0/32bit
-
-Tags: 3.0.7-alpine, 3.0-alpine
-Architectures: amd64
-GitCommit: fa0e115934f352cbfa8d0d471b43a18e865e28b3
-Directory: 3.0/alpine
-
-Tags: 3.0.504-windowsservercore, 3.0-windowsservercore
-Architectures: windows-amd64
-GitCommit: ad72d7a7f3c05a9b658ec64894d4193c89bba01b
-Directory: 3.0/windows/windowsservercore
-Constraints: windowsservercore
-
-Tags: 3.0.504-nanoserver, 3.0-nanoserver
-Architectures: windows-amd64
-GitCommit: ad72d7a7f3c05a9b658ec64894d4193c89bba01b
-Directory: 3.0/windows/nanoserver
-Constraints: nanoserver
-
-Tags: 3.2.9, 3.2, 3, latest
+Tags: 3.2.9, 3.2, 3
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le, s390x
 GitCommit: f131897d1f6305b8f6ebae695344f19c523e2c65
 Directory: 3.2
 
-Tags: 3.2.9-32bit, 3.2-32bit, 3-32bit, 32bit
+Tags: 3.2.9-32bit, 3.2-32bit, 3-32bit
 Architectures: amd64
 GitCommit: f131897d1f6305b8f6ebae695344f19c523e2c65
 Directory: 3.2/32bit
 
-Tags: 3.2.9-alpine, 3.2-alpine, 3-alpine, alpine
+Tags: 3.2.9-alpine, 3.2-alpine, 3-alpine
 Architectures: amd64
 GitCommit: f131897d1f6305b8f6ebae695344f19c523e2c65
 Directory: 3.2/alpine
 
-Tags: 3.2.100-windowsservercore, 3.2-windowsservercore, 3-windowsservercore, windowsservercore
+Tags: 3.2.100-windowsservercore, 3.2-windowsservercore, 3-windowsservercore
 Architectures: windows-amd64
 GitCommit: f131897d1f6305b8f6ebae695344f19c523e2c65
 Directory: 3.2/windows/windowsservercore
 Constraints: windowsservercore
 
-Tags: 3.2.100-nanoserver, 3.2-nanoserver, 3-nanoserver, nanoserver
+Tags: 3.2.100-nanoserver, 3.2-nanoserver, 3-nanoserver
 Architectures: windows-amd64
 GitCommit: f131897d1f6305b8f6ebae695344f19c523e2c65
 Directory: 3.2/windows/nanoserver
 Constraints: nanoserver
```

Comment:
(IMO, it'd be weird as a user for `redis:latest` to give me Redis 4.x but `redis:nanoserver` to give me 3.x, if both are still claimed to be supported -- I think https://github.com/MicrosoftArchive/redis/issues/525 makes it pretty clear that the future of Redis on Windows is tenuous at best.  Maybe one of our RedisLabs maintainers could comment on whether the Windows variants should simply go away at this point?)